packages/widget/src/components/TokenSelector.tsx

```diff
@@ -230,6 +230,7 @@ class TokenSelector extends React.Component<TokenSelectorProps, TokenSelectorSta
         <span className={cx('name')}>
           {selectedToken.symbol} <small>{selectedToken.name}</small>
         </span>
+        <span className={cx('caret')}>▾</span>
```

Comment:
only show caret if `!isSingle` (that is, is we have more than one token)

code would be something like:
```jsx
{!isSingle && (<span className={cx('caret')}>▾</span>)}
```
```

Reply:
Done.